pandas/core/algorithms.py

```diff
@@ -508,6 +512,7 @@ def duplicated(values, keep='first'):
     duplicated : ndarray
     """
 
+    values = com._asarray_tuplesafe(values)
```

Comment:
Yeah, that's perfectly reasonable.  I'll update the documentation to make that clear to avoid confusion.